xarray/test/test_dataset.py

```diff
@@ -1849,6 +1849,75 @@ def test_convert_dataframe_with_many_types_and_multiindex(self):
             expected = expected.drop(['h'], axis=1)
         assert roundtripped.equals(expected)
 
+    def test_to_and_from_dict(self):
+        """
+        <xarray.Dataset>
```

Comment:
We normally don't write docstrings for test methods. Put this in a multi-line comment instead.